src/System.Reflection.MetadataLoadContext/src/Configurations.props

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <PackageConfigurations>
+      netstandard;
+      netcoreapp
+    </PackageConfigurations>
+    <BuildConfigurations>
+      netstandard;
```

Comment:
In the review we said we'd make a netstandard ref assembly but implement net4x and netcoreapp with netstandard implementation throwing platformnotsupported. This would eliminate the type delegator hack.

Reply:
That said, if the hack works, I don't really have an issue with it.

Reply:
The hack does work, although not as efficient from directly deriving from TypeInfo because of an extra (unused) field in TypeDelegator.

IMO keeping it in netstandard would prevent downstream issues with netstandard consumers from having to deal with a platform-specific version of S.R.MetadataLoadContext.dll or from having it not work via PNSE. If we change netstandard in the future to expose a protected ctor on TypeInfo that would also address this in the future.

Reply:
@ericstj is a netstandard assembly preferred (in general) when we only have scenarios for using it with netfx, or does it make sense just to target netfx instead?

In this case, targeting netfx makes a case for removing a hack that is present in netstandard, but not sure what we would be giving up by not also supporting netstandard or making that a PNS assembly. For example, a third party assembly or tooling like Xamarin may only want to target netstandard for simplicity. I suppose we could support all three (netcoreapp, standard and netfx) if we had to.